packages/wonder-blocks-testing/src/harness/adapters/__tests__/data.test.js

```diff
@@ -10,57 +10,57 @@ describe("WonderBlocksData.adapter", () => {
         const children = <div>CONTENT</div>;
 
         // Act
-        const wrapper = mount(Data.adapter(children, []), {
-            includeDefaultTestHarness: false,
-        });
+        render(Data.adapter(children, []));
 
         // Assert
-        expect(wrapper).toHaveHTML("<div>CONTENT</div>");
+        expect(screen.getByText("CONTENT")).toBeInTheDocument();
     });
 
-    it("should render children within InterceptRequests when dataIntercepts configured", () => {
+    it("should render children within InterceptRequests when dataIntercepts configured", async () => {
         // Arrange
-        const children = <div>CONTENT</div>;
+
+        const TestFixture = () => {
+            const [result] = useCachedEffect("ID", jest.fn());
```

Comment:
**suggestion:** This works well enough. It would involve less "infrastructure" to use `useRequestInterception` although that would be more work here in the test, I think. This seems good enough to me (as long as the shared cache is getting cleared between tests).